src/main/java/com/example/shorturl/ShortenUrlApplication.java

```diff
@@ -1,9 +1,12 @@
 package com.example.shorturl;
 
+import com.example.shorturl.dao.UserRepository;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication
+@EnableJpaRepositories(basePackageClasses = UserRepository.class)
```

Comment:
Are you sure, if we need to use this annotation?

Reply:
I saw that it was added in a tutorial, but I checked and it is use to configure multiple jpa entities. It is not really necessary, I will remove it